src/shared/components/QuickLaneContent/QuickLaneContent.tsx

```diff
@@ -0,0 +1,235 @@
+import { Alert, Spacer, Tabs } from '@viaa/avo2-components';
+import { type Avo, PermissionName } from '@viaa/avo2-types';
+import { noop } from 'lodash-es';
+import React, { type FC, useEffect, useState } from 'react';
+
+import { PermissionService } from '../../../authentication/helpers/permission-service';
+import useTranslation from '../../../shared/hooks/useTranslation';
+import withUser, { type UserProps } from '../../hocs/withUser';
+import { useTabs } from '../../hooks/useTabs';
+import { ToastService } from '../../services/toast-service';
+
+import { isShareable } from './QuickLaneContent.helpers';
+import { type QuickLaneContentProps } from './QuickLaneContent.types';
+import QuickLaneContentPublicationTab from './QuickLaneContentPublicationTab';
+import QuickLaneContentSharingTab from './QuickLaneContentSharingTab';
+import './QuickLaneContent.scss';
+
+// State
+
+const QuickLaneContentTabs = {
+	publication: 'publication',
+	sharing: 'sharing',
+};
```

Comment:
Isn't an enum more suitable?
```suggestion
enum QuickLaneContentTabs {
	publication = 'publication',
	sharing = 'sharing',
}
```